palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/service/ResultAggregationService.java

```diff
@@ -119,7 +123,7 @@ private void auditRequestReceivedException(final RegisterDataRequest request, fi
                         .withContext(request.getContext())
                         .withException(ex)
                         .withServiceClass(serviceClass);
-        LOGGER.debug("Error handling: " + ex.getMessage());
+        LOGGER.debug("Auditing request exception: \n\t{}\n\t{}", request, ex);
```

Comment:
Here we are logging an exception event, we will only be here if an exception has occurred so we will certainly want to see that meta data whatever the level so ERROR level is applicable.